src/app/pages/Home/Home.tsx

```diff
@@ -40,24 +59,58 @@ export default function Home(): JSX.Element {
   );
 
   const searchedCard = filteredCards.filter(
-    (card) => card.title === searchValue
+    (card) => card.title.toLowerCase() === searchValue.toLowerCase()
   );
 
-  const [modalToggle, setModalToggle] = useState(false);
-  const [deleteCard, setDeleteCard] = useState<CodeCards>({
-    title: '',
-    content: '',
-    language: '',
-    collections: [],
-  });
-
   function handleDeleteClick(card: CodeCards) {
     removeCodeCard(card);
-    setModalToggle(false);
+    setDeleteModalToggle(false);
+  }
+  function handleChange(value: string) {
+    setContent(value);
+  }
+  // EDIT CARD ----------------------------------------------------------<Start>
+  function handleEditClick() {
+    const editCard = {
+      title: newTitle,
+      content: content,
+      language: deleteCard.language,
+      collections: deleteCard.collections,
+    };
+    console.log(editCard);
+    console.log(deleteCard);
+    editCodeCard(deleteCard, editCard);
+
+    setEditModalToggle(false);
   }
+  // EDIT CARD ----------------------------------------------------------<END>
+  // BAUARBEITEN ----------------------------------------------------------<Start>
+  const [showCollection, setShowCollection] = useState<string>('');
+  const [collectionTitle, setCollectionTitle] = useState<string>('');
+  function buildCollection(
+    cardCollection: { children: string; language: string }[]
+  ) {
+    const useCollections = cardCollection.map((collection) => {
+      return {
+        children: collection.children,
+        language: collection.language,
+        onClick: () => {
+          setShowCollection(collection.children),
+            setShowCollectioModalToggle(true),
+            setCollectionTitle(collection.children);
+        },
+      };
+    });
+    return useCollections;
+  }
+  console.log(showCollection);
+  //BAUSTELLE -----------------------------------------------------------<END>
```

Comment:
🚧 shouldn't be in the final code :)

Reply:
Will be overworked in next branch (its like a book mark for me)